local-proxy/src/server.ts

```diff
@@ -91,13 +93,13 @@ const app = new Koa();
 app.use(bodyParser({ enableTypes: ['json'], strict: false }));
 const router = new KoaRouter();
 
-const tmpDir = mkdtempSync(resolvePath(basePath, '..', '.reshuffle_local_proxy_'));
+const genDir = mkdtempSync(resolvePath(tmpDir, 'local_proxy_'));
```

Comment:
This has an issue, that now it's difficult to reliably access a file from `src` in code that is in `backend`, since it will have a different relative path.

This might not be a problem yet, since none of our apps currently do this, but to allow it I think we can add a new variable `RESHUFFLE_PROJECT_ROOT` or something similar.

Reply:
Good catch!
I'm not sure we want to let users import from `src`, we don't even upload it today as part of the bundle.

https://github.com/reshufflehq/reshuffle/blob/2f5854729d75bd2c71d14d2f5652494d17a0713c/cli/src/commands/deploy.ts#L73